releasenotes/notes/set-content-lenght-flare-3b220d6a30cef1b8.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Ensure flares with the new low memory footprint process can be uploaded.
```

Comment:
This should probably be moved to the dca release not as this wasn't released on the agent yet